app/forms/idv/api_image_upload_form.rb

```diff
@@ -204,14 +209,15 @@ def as_readable(image_key)
       end
     end
 
-    def update_analytics(client_response)
+    def update_analytics(client_response, vendor_request_time_in_ms)
       add_costs(client_response)
       update_funnel(client_response)
       analytics.idv_doc_auth_submitted_image_upload_vendor(
         **client_response.to_h.merge(
           client_image_metrics: image_metadata,
           async: false,
           flow_path: params[:flow_path],
+          vendor_request_time_in_ms: vendor_request_time_in_ms,
```

Comment:
also I *think* that the Timer class in ruby returns floating point seonds, not milliseconds https://ruby-doc.org/core-2.6.3/Time.html#method-i-to_f

```suggestion
          vendor_request_time_in_ms: vendor_request_time_in_ms * 1000,
```

Reply:
I think we're already doing that in the timer helper I'm using: https://github.com/18F/identity-idp/blob/main/app/jobs/job_helpers/timer.rb#L17

Reply:
d'oh I was literally looking at that line of code... that I wrote.... it's definitely Friday brain today